src/routers/AppRouter.tsx

```diff
@@ -162,23 +162,32 @@ const AktivBrukerRouter = (): ReactElement => {
           <Route
             path={manglendeMedvirkningPath}
             element={
-              <ManglendeMedvirkningSide>
-                <ManglendeMedvirkning />
-              </ManglendeMedvirkningSide>
+              <NotificationProvider>
+                <Outlet />
```

Comment:
Hva er outlet?

Reply:
Det er en slags placeholder som rendrer alle child-routes av `manglendeMedvirkningPath` som ligger definert under her. https://reactrouter.com/en/main/components/outlet. 
Bruker det her for å kunne dele tilstand på tvers av alle routes under manglendemedvirkning-siden. `<NotificationProvider`> har tilstand som vi ønsker å sette/vise på ulike manglendemedvirkning-sider, og siden `<Outlet />` wrappes i `<NotificationProvider`> så vil alle child-routes kunne bruke tilstand definert i `<NotificationProvider`>.
Fant en artikkel om `useOutletContext` også https://medium.com/@jpwesley34/using-usecontext-and-useoutletcontext-e59ee3de3e46, mulig vi kunne forenklet til å bruke det :thinking:

Reply:
Spennende! Så alle komponenter som renderes på den routen vil arve egenskapene til `NotificationProvider`?

Reply:
Ja, de vil kunne ta i bruk contexten som `NotificationProvider` tilbyr, dvs kunne hente ut og sette `Notification`.